button/button.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Compopnent Library</title>
+    <link href="../style.css" rel="stylesheet" />
+    <link href="button.css" rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
+  </head>
+  <body>
+    <nav class="nav-container">
+      <a class="home nav-item" href="/">home</a>
+      <a class="docs nav-item" href="/">docs</a>
+    </nav>
+
+    <div class="page-flex-container">
+      <aside class="page-flex-item sidebar">
+        <h1 class="heading-sidebar">Components</h1>
+        <a class="sidebar-item" href="/avatar/avatar.html">Avatar</a>
+        <a class="sidebar-item" href="/alert/alert.html">Alert</a>
+        <a class="sidebar-item" href="/badge/badge.html">Badge</a>
+        <a class="sidebar-item current-sidebar-item" href="/button/button.html">Button</a>
+        <a class="sidebar-item" href="/">Card</a>
+        <a class="sidebar-item" href="/">Image</a>
+        <a class="sidebar-item" href="/">Input</a>
+      </aside>
+
+      <main class="page-flex-item main-content">
+        <h1 class="heading">Button</h1>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut at orci at
+          justo imperdiet mollis nec vitae erat. Curabitur commodo sapien non
+          diam pharetra vehicula. Praesent efficitur neque nec eros volutpat
+          facilisis.
+        </p>
+         
+        <h2 class = "heading">Primary Button</h2>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut at orci at
+            justo imperdiet mollis nec vitae erat. Curabitur commodo sapien non
+            diam pharetra vehicula. Praesent efficitur neque nec eros volutpat
+            facilisis.
+          </p>
+
+        <div class="btn-container">
+            
+          <button class="btn-item btn-primary btn-flex btn-primary-1">Primary</button>
```

Comment:
Try writing more descriptive class name.

Reply:
Agreed to humza also unable to comprehend where its just color change or something different.

Reply:
I have made the changes.Thanks for review